src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderSentenceRepository.java

```diff
@@ -9,6 +9,6 @@
 
 public interface OffenderSentenceRepository extends CrudRepository<OffenderSentence, PK> {
     @EntityGraph(value = "sentence-entity-graph")
-    List<OffenderSentence> findByOffenderBooking_BookingId_AndCalculationType_CalculationTypeNotLikeAndCalculationType_CategoryNot(
-        Long bookingId, String calculationType, String category);
+    List<OffenderSentence> findByOffenderBooking_BookingId_AndCalculationType_CategoryNot(
```

Comment:
Approved - as this is just a temporary change to help diagnose an issue with how the DB query is generated